source/common/http/header_map_impl.cc

```diff
@@ -285,6 +285,17 @@ template <> HeaderMapImpl::StaticLookupTable<ResponseTrailerMap>::StaticLookupTa
   finalizeTable();
 }
 
+absl::string_view HeaderMapImpl::delimiterByHeader(const LowerCaseString& key,
+                                                   bool correctly_coalesce_cookies) {
+  // TODO(wbpcode): 'correctly_coalesce_cookies' feature is enabled by default and is allowed to be
+  // disabled via runtime. But I doubt if any user will actually disable it. The comma separator is
+  // obviously problematic for cookies. Maybe we can consider removing it.
```

Comment:
It'll be removed in a few months per the process documented in CONTRIBUTING.md so I think you can take this TODO out.

Reply:
get it.